### ISSUE TITLE
Coromon (1218210)

### ISSUE THREAD

**Issue by Preylien** [ValveSoftware/Proton#5736]
# Compatibility Report
- Name of the game with compatibility issues: Coromon
- Steam AppID of the game: 1218210

## System Information
- GPU: Nvidia RTX 2080ti
- Driver/LLVM version: nvidia 470.86
- Kernel version: 5.16.11-76051611-generic
- Link to full system information report as [Gist](https://gist.github.com/Preylien/d2fefc4b2ea47b90fdd4b0b848a8b488):
- Proton version: Experimental, 7.0-1, 4.11-13 

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
App crashes on startup with popup when using Proton Experimental or 7.0-1:

A JavaScript error occurred in the main process
```
Uncaught Exception:
Error: EINVAL: invalid argument, uv_pipe_open
  at new Socket (net.js:330:26)
  at createWritableStdioStream (internal/bootstrap/switches/is_main_thread.js:72:18)
  at process.getStdout [as stdout] (internal/bootstrap/switches/is_main_thread.js:122:12)
  at console.get(internal/console/constructor.js:204:42)
  at console.value(internal/console/constructor.js:322:50)
  at console.log(internal/console/constructor.js:358:61)
  at App.<anonymous>(Z:\home\username\storage\games\steam\steamapps\common\Coromon\FreedomLauncher\resources\app.asar\dist\index.js:1:8254)
  at App.emit(events.js:315:20)

```

Running the game on Proton 4.11-13 a window, Freedom Games Launcher, will appear and have a black screen

## Reproduction

Open game using Proton experimental, or 7.0-1, game will crash with a popup.

Running the game on Proton 4.11-13 a window, Freedom Games Launcher, will appear and have a black screen

**Comment by Preylien** [comment]
I forgot to mention. This game had a demo that did work when I played it, last year I think.
That demo did work, though I have no idea what version of Proton it used.

I wonder if "Freedom Games Launcher" was not included in the demo, and that is where it's getting hung up compared to then? I can't really validate that info though.

**Comment by Preylien** [comment]
Seems like an update this afternoon has fixed it for Proton 7.0-1. I was able to get past the main menu. I'll close this as it seems to be working :+1:
